api-myhome/src/controllers/realtor.controller.js

```diff
@@ -91,6 +95,21 @@ class RealtorController {
       next(err);
     }
   }
+
+  async addReview(req, res, next) {
+    const { realtorId } = req.params;
+    const { rating, comment, userId } = req.body;
```

Comment:
aca es como que desarmas el body y despues lo volves a armar. si en el body te viene la review capaz podes hacer mas facil esto
```suggestion
    const { review } = req.body;
```

Reply:
idem comment arriba